### PR TITLE
feat: NodeUpdate needs only admin key to sign

### LINKED ISSUE #16493: NodeUpdate needs only admin key to sign.
Revert back to original adminKey sign required for NodeUpdate.

### PR DESCRIPTION
**Pull request by iwsimon**
**Description**:

**Related issue(s)**:

Fixes #16493 

**Notes for reviewer**:

**Checklist**

- [ ] Documented (Code comments, README, etc.)
- [ ] Tested (unit, integration, etc.)

### CODE REVIEW

File: hapi/hedera-protobufs/services/address_book_service.proto [hunk 1]
```diff
@@ -91,18 +91,20 @@ import "transaction.proto";
...
- *      network.
+ *       - The node operator SHALL submit the transaction to the
+ *         network.  Hedera council approval SHALL NOT be sought for this
+ *         transaction
+ *    - If the Hedera council representative creates the transaction
+ *       - The Hedera council representative SHALL arrange for council members
+ *         to review and sign the transaction.
+ *       - Once sufficient council members have signed the transaction, the
+ *         Hedera council representative SHALL submit the transaction to the
+ *         network.
```
Comment (jsync-swirlds) [inline hapi/hedera-protobufs/services/address_book_service.proto:102]: This block of text can be removed.

File: hapi/hedera-protobufs/services/address_book_service.proto [hunk 2]
```diff
@@ -91,18 +91,20 @@ import "transaction.proto";
...
  *   software SHALL
- *    - Validate the threshold signature for the Hedera governing council
- *    - Validate the signature of the active `key` for the account to be
- *      assigned as the "node account".
+ *    - If the transaction is signed by the active `key` for the node account
+ *       - Validate the signature of the active `key` for the account assigned
+ *         as the "node account".
  *    - If the transaction modifies the value of the "node account",
  *       - Validate the signature of the _new_ `key` for the account to be
  *         assigned as the new "node account".
```
Comment (jsync-swirlds) [inline hapi/hedera-protobufs/services/address_book_service.proto:110]:
```suggestion
 *    - If the transaction modifies the value of the "node account",
 *       - Validate the signature of the active `key` for the account
 *         assigned as the _current_ "node account".
 *       - Validate the signature of the active `key` for the account to be
 *         assigned as the _new_ "node account".
```

File: hapi/hedera-protobufs/services/address_book_service.proto [hunk 3]
```diff
@@ -91,18 +91,20 @@ import "transaction.proto";
  *       - If the transaction changes the value of the "node account" the
  *         node operator MUST _also_ sign this transaction with the active `key`
  *         for the account to be assigned as the new "node account".
```
Comment (jsync-swirlds) [inline hapi/hedera-protobufs/services/address_book_service.proto:93]: This block has a duplicate below, and can be removed from here.

**Review by jsync-swirlds — COMMENTED** [review]
Lines 86-87 should also be updated to remove reference to the council creating the transaction and the "if node operator created", as the node operator will always create.

File: hapi/hedera-protobufs/services/address_book_service.proto (hunk 1 above)
Comment (iwsimon) [reply hapi/hedera-protobufs/services/address_book_service.proto]: Removed.

File: hapi/hedera-protobufs/services/address_book_service.proto (hunk 2 above)
Comment (iwsimon) [reply hapi/hedera-protobufs/services/address_book_service.proto]: Updated.

File: hapi/hedera-protobufs/services/address_book_service.proto (hunk 3 above)
Comment (iwsimon) [reply hapi/hedera-protobufs/services/address_book_service.proto]: Removed.

File: hapi/hedera-protobufs/services/address_book_service.proto [hunk 4]
```diff
@@ -83,31 +83,19 @@ import "transaction.proto";
...
- *         transaction
- *    - If the Hedera council representative creates the transaction
- *       - The Hedera council representative SHALL arrange for council members
- *         to review and sign the transaction.
- *       - Once sufficient council members have signed the transaction, the
- *         Hedera council representative SHALL submit the transaction to the
- *         network.
+ * - The node operator SHALL create an `updateNode` transaction.
+ *    - The node operator MUST sign this transaction with the active `key`
+ *      for the account assigned as the current "node account".
```
Comment (jsync-swirlds) [inline hapi/hedera-protobufs/services/address_book_service.proto:88]:
```suggestion
 *      assigned as the `admin_key`.
```

Comment (iwsimon) [reply hapi/hedera-protobufs/services/address_book_service.proto]: Updated.

**Review by jsync-swirlds — APPROVED** [review]

**Review by tinker-michaelj — APPROVED** [review]
LGTM, tyvm @iwsimon !

**Pull request merged**